cloudbuild/run-tests.sh

```diff
@@ -101,9 +108,16 @@ main() {
   cd /init-actions
   configure_gcloud
   configure_gcloud_ssh_key
-  initialize_git_repo
-  determine_tests_to_run
-  run_tests
+  if [[ $RUN_ALL_TESTS=="true" ]]; then
+    # Run periodic
+    initialize_git_repo "false"
+    bash cloudbuild/periodic/run-all-tests.sh
```

Comment:
Seems like we can reuse `run_tests` function by modifying it to disable logic not needed for periodic tests and calling it with `true` to make it always succeed:
```bash
run_tests || true
```

Reply:
Done.